activerecord/test/cases/arel/helper.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "active_support"
```

Comment:
`require "active_support/test_case"` like here:

https://github.com/rails/rails/blob/a42fb70a152ee05725685a8f68e3e8e2ddda4b36/activerecord/test/cases/test_case.rb#L3

Reply:
Thanks for the review. Updated.